backend/src/main/scala/com/softwaremill/bootzooka/user/UserService.scala

```diff
@@ -144,9 +142,10 @@ class UserService(
       _ <- validateNewPassword()
       _ = logger.debug(s"Changing password for user: $userId")
       _ <- userModel.updatePassword(userId, User.hashPassword(newPassword))
-      confirmationEmail = emailTemplates.passwordChangeNotification(user.login)
-      _ <- emailScheduler(EmailData(user.emailLowerCased, confirmationEmail))
-    } yield ()
+    } yield {
+      val confirmationEmail = emailTemplates.passwordChangeNotification(user.login)
+      emailScheduler(EmailData(user.emailLowerCased, confirmationEmail))
```

Comment:
This effect needs to be sequenced with the others - I'm surprised this worked (did tests pass)? Anyway, fixing